source/core/Array.cpp

```diff
@@ -1243,7 +1243,7 @@ VIREO_FUNCTION_SIGNATURE7(ArrayDeleteND, TypedArrayCoreRef, StaticType, void,
 
     IntIndex startIndex = offset > 0 ? offset : 0;
     IntIndex endIndex = (offset +
```

Comment:
Could you break apart this statement into variables so it's easier to read?

Reply:
Done.